charts/hoprd-operator/templates/crossplane.yaml

```
@@ -0,0 +1,84 @@
+    member: allUsers
+    role: roles/storage.objectViewer
+  providerConfigRef:
+    name: {{ .Values.operator.crossplane.provider.upbound }}
+---
+apiVersion: storage.gcp.upbound.io/v1beta1
+kind: BucketIAMMember
+metadata:
+  name: hoprd-operator-bucket-write
+spec:
+  forProvider:
+    bucket:  {{ .Values.operator.crossplane.bucket.name }}
```

yamllint (1.35.1)

[warning] too many spaces after colon

(colons)